### ISSUE TITLE
Browser Testing & Accessibility Testing 

### ISSUE THREAD

**Issue by pamxla** [Sara-Elmelegy/web-design-2-independent-project#4]
Hi! 
I've checked your website and it looks good, all ur nav links work as well as your "explore" buttons. When testing for the different screen sizes, I noticed some of your buttons were going over the image above which made it hard to see. In the small screen size, your nav links are very close together so I think they should have some space or be placed differently.
![Screen Shot 2021-12-10 at 12 10 00 PM](https://user-images.githubusercontent.com/77292983/145613881-2558cd05-d97e-4ef3-95df-1bcd685ea1b7.png)
For the full-screen size of your website home page, the h2's were close together to the images, so I thought maybe they could have some space. Also when I go to the Photography, Paintings, and Graphic Design pages, I noticed that there is no nav, so I think you should add the nav into those pages so the viewer can go back to the home page easily.

**Accessibility** 
Your performance and accessibility are good, it just needs some fixes with your aria labels according to the chrome extension. I was also testing the voiceover and it didn't read the nav and most of the content on your website. You are also missing skip links. Overall, everything looks good, it just needs a few fixes. Good work! and your art pieces are beautiful! 😄 
![Screen Shot 2021-12-10 at 12 18 51 PM](https://user-images.githubusercontent.com/77292983/145615204-1946d8f0-b89b-4e43-a483-fd5e578876d2.png)

**Comment by pamxla** [comment]
I forgot to mention
I ran your website through WAVE https://wave.webaim.org/report#/https://tringuyen479.github.io/web-design-2-independent-project/ and only found some tinny alerts saying you have some empty links.
![Screen Shot 2021-12-10 at 1 09 35 PM](https://user-images.githubusercontent.com/77292983/145621525-75075044-6b44-4091-8014-75942bfdfccd.png)
You should also check here https://www.webpagetest.org/result/211210_BiDc6W_fc39423285eeffb28a05e9a4e8560917/
